src/vs/editor/browser/controller/editContext/native/nativeEditContext.ts

```diff
@@ -49,6 +50,8 @@ export class NativeEditContext extends AbstractEditContext {
 
 	private readonly _focusTracker: FocusTracker;
 
+	private _selectionChangeListener: IDisposable | undefined;
```

Comment:
`MutableDisposable` is a much nicer pattern for this type of thing.

Reply:
thanks for the suggestion ^^